homework/01LessonOne.js

```diff
@@ -119,32 +121,37 @@ var emptyString = '';
 // Problem 17:
 // What is the value of !!bob
 
-var notNotBob;
+var notNotBob = true;
 
 // Problem 18:
 // What is the value of !!emptyString
 
-var notNotEmptyString;
+var notNotEmptyString = false;
 
 // Problem 19:
 // What is the value of !null
 
-var notNull;
+var notNull = true;
 
 // Problem 20:
 // What is the value of !!undefined
 
-var notNotUndefined;
+var notNotUndefined = false;
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
-// Explain truthiness and falsiness in your own words. Provide an example for each.
+// Explain truthiness and falsiness in your own words. Provide an example for
+//each.
 //
 // Your Answer Goes Here:
-//
+//Truthiness is a concept used to describe emotionally positive values.
+//A value such as "string" or 5 is "truthy" because it is emotionally positive,
+//in addition to the boolean operator true.
+//Falsiness is the converse of truthiness.  It means the value is emotionally
+//negative or is undefined.  E.g. 0, null, false.
```

Comment:
Not sure about the "emotionally negative" or "emotionally positive" stuff.

Truthy values are treated as true. Falsy values are treated as false.